src/schema/artwork/contextGrids/PartnerArtworkGrid.ts

```diff
@@ -0,0 +1,64 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import {
+  ContextGridType,
+  formDefaultGravityArgs,
+} from "schema/artwork/contextGrids"
+import { artworkConnection } from "schema/artwork"
+import { connectionFromArraySlice } from "graphql-relay"
+import { pageable } from "relay-cursor-paging"
+
+export const PartnerArtworkGridType = new GraphQLObjectType<
+  { partner: any; artwork: any },
+  any
+>({
+  name: "PartnerArtworkGrid",
+  interfaces: [ContextGridType],
+  fields: () => ({
+    title: {
+      type: GraphQLString,
+      resolve: ({ partner }) => {
+        if (!partner) return null
+        return `Other works from ${partner.name}`
+      },
+    },
+    ctaTitle: {
+      type: GraphQLString,
+      resolve: ({ partner }) => {
+        if (!partner) return null
```

Comment:
i wonder if we need to do something about `if (!partner) return null` and if its even needed. seems to be repeated in few places and im not sure in what case we land in there.

Reply:
I'd like for the resolver to not have to care about from where it's being created (i.e. the context grid or otherwise), so in that sense there's no requirement that `partner` exists at this point.